src/emucore/Console.cxx

```diff
@@ -463,8 +463,8 @@ void Console::initializeAudio()
 */
 void Console::fry() const
 {
-  for (int ZPmem=0; ZPmem<0x100; ZPmem += rand() % 4)
-    mySystem->poke(ZPmem, mySystem->peek(ZPmem) & (uInt8)rand() % 256);
+  for (int ZPmem=0; ZPmem<0x100; ZPmem += randNumGen.next() % 4)
+    mySystem->poke(ZPmem, mySystem->peek(ZPmem) & (uInt8)randNumGen.next() % 256);
```

Comment:
I think what the example is showing is that std::cout prints out uint8_t's as characters (rather than integers) :) Note that the cast has higher precedence than modulo.

But yes, since it was already in the code there's no harm in leaving it there, but it is very strange.